django_app/frontend/style.css

```diff
@@ -8433,7 +8433,6 @@ body:has(.iai-environment-warning) main {
 .iai-chat-bubble__route {
   font-size: 0.625rem;
   line-height: 0.825rem;
-  display: none;
```

Comment:
This was removed as part of https://github.com/i-dot-ai/redbox-copilot/pull/698 and I think might have got re-added during a rebase